cmake/EthUtils.cmake

```diff
@@ -62,3 +62,13 @@ macro(eth_add_test NAME)
 
 endmacro()
 
+# Creates C resources file from files
+function(eth_add_resources TARGET RESOURCE_FILE)
+
+	add_custom_target("${TARGET}.resources"
```

Comment:
Consider using MAIN_DEPENDENCY/DEPENDS so that it would only update resources when required